ee/clickhouse/views/experiments.py

```diff
@@ -159,7 +159,13 @@ class ClickhouseExperimentsViewSet(StructuredViewSetMixin, viewsets.ModelViewSet
     permission_classes = [IsAuthenticated, ProjectMembershipNecessaryPermissions, TeamMemberAccessPermission]
 
     def get_queryset(self):
-        return super().get_queryset()
+        filters = self.request.GET.dict()
+        if "user" in filters:
+            return super().get_queryset().filter(created_by=self.request.user)
+        if "archived" in filters:
```

Comment:
I just added this to demonstrate frontend functionality for it, this will probably have to change once we make updates to our experiments model to remove `parameters`